teams/vsp/teams/operations/daytime-oncall-support-role.md

```diff
@@ -54,14 +56,16 @@ Your Github team notifications and [#vsp-operations](https://dsva.slack.com/chan
 
 ### Support requests
 
-Create an issue describing the request in GitHub and link it to one of the two super-epics. Assign it to yourself. Track any work done with comments. Only track support that takes more than 15 mins and if it is a PR that takes under 15 mins please link to the super epic. If the support work seems to be taking you down a long road, make a note around this and bring to the team to scope out into a project. Unless an emergency doesn't spend more than 2 hours on any given support work. Summarize both failed attempts and final solutions so the issue can be referred to again in the future if necessary. If other team members are involved, add them to the ticket.
+Create an issue describing the request in GitHub and link it to one of the two Super Epics. Assign it to yourself. Track any work done with comments. Only track support that takes more than 15 mins and if it is a PR that takes under 15 mins please link to the super epic. If the support work seems to be taking you down a long road, make a note around this and bring to the team to scope out into a project. Unless an emergency, don't spend more than 2 hours on any given support work. Summarize both failed attempts and final solutions so the issue can be referred to again in the future if necessary. If other team members are involved, add them to the ticket.
 
 ### Working on Postmortem action items
-In Zenhub there is a label entitled "Postmortem Action Item". These are tasks that have been identified after an incident's postmortem. The oncall Dev should work on these items If there is no active support work or incident. If there is another dev working on the Postmortem Action Items connect with them to try and see how to help get that work completed. 
+In Zenhub there is a label entitled "Postmortem Action Item". These are tasks that have been identified after an incident's postmortem. The oncall Dev should work on these items If there is no active support work or incident. If there is another dev working on the Postmortem Action Items, connect with them to try and see how to help get that work completed.
 
 ### Supporting and Monitoring Deployments
 If deploys fail and there is no direct owner, take some time to try and find the cause of the failed deploy. If you do decide to look into a failed deployment make sure to start a thread under it and let others know. 
 
+Can we list some Slack channels or links to Jenkins here that Oncall should monitor?
```

Comment:
Question here. 

Maybe the main oncall doc would be a better place to call them out?
https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/tree/master/OnCall

Reply:
Those channels are captured in https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsp/teams/operations/daytime-oncall-support-role.md